README.md

```diff
@@ -61,3 +61,28 @@ For more information, see [Kubemacpool Opt-Modes](doc/opt-modes.md).
 Part of the Kubemacpool functionality is implemented as a [kubernetes mutating admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook) 
 to ensure that the MAC address is assigned before VirtualMachine is created.
 This webhook uses self signed certificate. For more information see [certificate handling](doc/certificate-handling.md).
+
+### Metrics 
+Kubemacpool [Prometheus](https://prometheus.io/) expose the following metric:
+- `kubevirt_kmp_duplicate_macs`
+
+  The metric is a Gouge, its incremented when Kubemacpool detects MAC address conflict by Kubevirt VirtualMachines.
+
+  A MAC conflict occur in the following scenarios:
+  - Multiple VMs use the same MAC address
+  - VM with multiple interfaces, where multiple interfaces use the same MAC address. 
```

Comment:
The scenario are not accurate, and suggest KMP doesn't do it's sole job correctly. :)

I would rephrase entirely:
```
When KMP starts to run, it maps all the existing MACs (VMs and pods) in the cluster and checks for duplicates that exist. If it finds one, it raises this metric.
```